scripts/mutect2_wdl/mutect2.json

```diff
@@ -17,5 +17,6 @@
   "Mutect2.dbsnp_index": "$__dbsnp_index__",
   "Mutect2.cosmic": "$__cosmic__",
   "Mutect2.cosmic_index": "$__cosmic_index__",
-  "Mutect2.picard_jar": "$__picard_jar__"
+  "Mutect2.picard_jar": "$__picard_jar__",
+  "Mutect2.run_orientation_bias_filter": "$__run_orientation_bias_filter__"
```

Comment:
If it is ultimately a boolean, please name it ``$__is_run_orientation_bias_filter__`` ...